hashicorp/react-components
# Bump `swingset` to enable fullscreen content area

🎟️ [Asana Task]()
🔍 [Preview Link](https://react-components-git-{branch-slug}-hashicorp.vercel.app)

---



## Description
[A recent change](https://github.com/hashicorp/swingset/pull/48) to `swingset` was shipped that enables fullscreen content areas. This PR bumps `swingset` to the latest version to enable fullscreen components in `react-components`.

## Validation Steps
- Navigate to [the preview](https://react-components-iy993ymj5-hashicorp.vercel.app/components/nextsteps)
- Click the fullscreen button in the top right and verify that the sidebar slides away
- Click the button again and verify that the sidebar slides back into view
- Navigate to [a component with the `isFullscreen=true` query param](https://react-components-iy993ymj5-hashicorp.vercel.app/components/callouts?isFullscreen=true)
- Verify that the page transitions to fullscreen

### PR Checklist 🚀

Items in this checklist may not may not apply to your PR, but please consider each item carefully.
